tests/snuba/api/endpoints/test_organization_metrics_meta.py

```diff
@@ -241,14 +241,14 @@ def test_multiple_projects(self):
             project_id=self.project.id,
         )
         url = reverse(
-            "sentry-api-0-organization-events-metrics-compatibility",
+            "sentry-api-0-organization-metrics-compatibility-sums",
             kwargs={"organization_slug": self.project.organization.slug},
         )
         response = self.client.get(url, format="json")
 
         assert response.status_code == 200, response.content
         # project 4 shouldn't show up in these sums
-        assert response.data["sum"]["metrics"] == 3
+        assert response.data["sum"]["metrics"] == 4
```

Comment:
Does the comment here need to be updated? I'm not entirely sure but this diff looks like it's a bug. It looks like project 4 is appearing in the sums when the comment says it wasn't

Is it because in these changes ignore DS compatibility?

Reply:
I'll update the comment, these tests were wrong cause I was testing the old endpoint by mistake